packages/blade/src/components/Form/Selector/SelectorInput.web.tsx

```diff
@@ -36,7 +36,7 @@ const getHoverStyles = ({
     borderColor: getIn(theme, backgroundColor),
     backgroundColor: getIn(theme, backgroundColor),
     transitionTimingFunction: theme.motion.easing.standard.effective as string,
-    transitionDuration: makeMotionTime(theme.motion.duration['2xquick']),
+    transitionDuration: (makeMotionTime(theme.motion.duration['2xquick']) as unknown) as string,
```

Comment:
Make motion time returns `string` on web & `number` on native but Styled components only takes strings. 

While type checking native will fail, thus we need `as unknown as string`

Reply:
This is quite confusing. Can we do without `makeMotionTime`? Let styled-components take care of things?

Reply:
Agreed, thoughts @chaitanyadeorukhkar ?

Reply:
Still going through the PR and building context. Will come back to this 😁

Reply:
@chaitanyadeorukhkar yes that's actually weird it should not throw when doing build. Since i'm ignoring the web files on native compilation. I'm investigating this.

Reply:
Oh damn. Got it. It was failing because we had `SelectorInput.d.ts` which was reexport the web file. 

export * from './SelectorInput.web';